numpy/ma/tests/test_core.py

```diff
@@ -447,6 +447,16 @@ def test_deepcopy(self):
         assert_equal(copied.mask, [0, 0, 0])
         assert_equal(a.mask, [0, 1, 0])
 
+    def test_format(self):
+        # Note: Python 3.6 and above fstring uses __format__, see PEP 498.
+        # Testing with underlying primitive.
+        a = array([0, 1, 2], mask=[False, True, False])
+        assert_equal(format(a), "[0 -- 2]")
+        assert_equal(format(masked), "--")
+        assert_equal(format(masked, ""), "--")
+        assert_equal(format(masked, " >5"), "   --")
+        assert_equal(format(masked, " <5"), "--   ")
```

Comment:
You'll need to remove these tests

Reply:
Ah ok, I do not mind not supporting them for now. So removal or error check with a comment that this could be changed in the future sounds good to me.

Reply:
Yeah, adding support for formatting is a big task I do not think we should be attempting to address in this bugfix PR.